components/user_settings/modal/index.ts

```diff
@@ -31,6 +36,8 @@ function mapStateToProps(state: GlobalState) {
         sendEmailNotifications,
         requireEmailVerification,
         collapsedThreads,
+        globalHeaderEnabled: getGlobalHeaderEnabled(state),
+        isContentChannelPreferences: ownProps.isContentChannelPreferences,
```

Comment:
This line is unnecessary, it should pass this through anyways.

Reply:
why do we need this check? It seems like the globalHeaderEnabled could be enough for the use case. Also we don't need to add it to the index.ts if its coming from ownProps

Reply:
@nevyangelova, @devinbinnie agree on the `ownProps` 

@nevyangelova When `globalHeaderEnabled` is on we can have 2 modals `Channel Preferences` or `Account Settings`. `isContentChannelPreferences`  flag determines which one to show.